service_container/alias_private.rst

```diff
@@ -185,11 +197,14 @@ or you decided not to maintain it anymore), you can deprecate its definition:
 
             <services>
                 <service id="app.mailer" alias="App\Mail\PhpMailer">
-                    <!-- this will display a generic deprecation message... -->
-                    <deprecated/>
-
-                    <!-- ...but you can also define a custom deprecation message -->
-                    <deprecated>The "%alias_id%" service alias is deprecated. Don't use it anymore.</deprecated>
+                    <!-- this outputs the following generic deprecation message:
+                         Since acme/package 1.2: The "app.mailer" service is deprecated. You should stop using it, as it will be removed in the future -->
```

Comment:
```suggestion
                         Since acme/package 1.2: The "app.mailer" service alias is deprecated. You should stop using it, as it will be removed in the future -->
```